packages/grant-explorer/src/features/common/ExpandableGrid.tsx

```diff
@@ -0,0 +1,24 @@
+import { Grid } from "@chakra-ui/react";
+import React from "react";
+
+interface ExpandableGridProps {
+  isOpen: boolean;
+  classNames?: string;
+  children: React.ReactNode;
+}
+
+export const ExpandableGrid: React.FC<ExpandableGridProps> = ({
```

Comment:
Is there a good reason to use `React.FC<Props>`?
Why not:
```tsx
export const ExpandableGrid = ({ isOpen, classNames, children }: ExpandableGridProps) => 
```

Reply:
Using the `React.FC` inference offers more clarity to the dev and organization to the code, and for me it's a matter of standards. 
No complains at all if you want to change to pure functions